src/netflow/templates.rs

```diff
@@ -95,7 +95,7 @@ impl TemplatesBox {
     }
 
     #[must_use]
-    pub(super) fn get(&self, key: &TemplateKey) -> Option<&Template> {
+    pub(crate) fn get(&self, key: &TemplateKey) -> Option<&Template> {
```

Comment:
Since this method is only shared within the netflow module, pub(super) seems more appropriate.